commons/src/main/java/io/hyscale/commons/utils/ResourceLabelBuilder.java

```diff
@@ -70,6 +70,18 @@
 
 		return labels;
 	}
+	
+    public static Map<ResourceLabelKey, String> setVersion(Map<ResourceLabelKey, String> labels, String version) {
```

Comment:
Why is this required ?

Reply:
While setting the resource label map in manifest context under ManifestContextBuilder.